src/routes/apiRoutes/api.route.ts

```diff
@@ -15,5 +15,6 @@ router.get("/claimsfeed2", claimsFeed);
 router.get("/node/:nodeId?", nodesGet);
 router.get("/my-node", getNodeForLoggedInUser);
 router.get("/report/:claimId?", claimReport);
+router.get("/subjets/", claimsOfSubject);
```

Comment:
lets change the path to 

/about/:subject

It woudl be fine also if we just use the /report/:claimId path, that is what it was for

and generate the subject url from the claimId

Reply:
we could generate the subject deterministically from the claimId
currently as
https://linkedtrust.us/claims/[claimId]
we might change this later but this would work right now